examples/pump-app/nrfconnect/main/PumpManager.cpp

```diff
@@ -50,7 +50,7 @@ void PumpManager::SetCallbacks(Callback_fn_initiated aActionInitiated_CB, Callba
 
 bool PumpManager::IsActionInProgress()
 {
-    return (mState == kState_StartInitiated || mState == kState_StartInitiated) ? true : false;
+    return (mState == kState_StartInitiated) ? true : false;
```

Comment:
Any reason not to do:
```suggestion
    return (mState == kState_StartInitiated);
```
?  Either way is fine, though.  Just wanted to understand what considerations there are here.

Reply:
Yeah, that ended up looking a bit stupid, I was to focused on just removing the identical condition that I did not see that 😅 

I found two other instances of this, so I "fixed" that as well.